### ISSUE TITLE
[LaLa flatpak Bug]: unable to launch trainers in Steam Deck gaming mode, but can launch in desktop mode.

### ISSUE THREAD

**Issue by wyyadd** [wyyadd/LaLa#15]
### Bug Description
As mentioned in the title, I can't launch trainers in Steam Deck gaming mode using flatpak version of LaLa, but can launch in desktop mode. However, when I use the AppImage version of LaLa, I don't encounter this issue.

### How to reproduce
- Switching to desktop mode, launch game and trainer. Trainer is launched successfully.
- Switching to gaming mode, launch game and trainer. Trainers won't start and no response when clicking "Launch".

### Related issue
I find a related issue at https://github.com/bottlesdevs/Bottles/issues/2020

If anyone knows the reason for this issue, I would appreciate some guidance. ：)

**Comment by ahjolinna** [comment]
hmm.. yes the AppImage "works"...but some have the flickering issue there so kinda hard to do the minimize thingy (have to use touchscreen and it goes so fast) ...sigh...oh well

about the flatpak issue, well only thing that makes SteamOS different from others when in GamingMode is that it uses [gamescope](https://wiki.archlinux.org/title/Gamescope) for wayland ...(desktop/KDE it still uses x-server) so there might be some issue with gamescope? ...I would start look into gamescope (developed mainly by valve)

found something like this: https://github.com/tinywrkb/org.freedesktop.Platform.VulkanLayer.GameScope 
their git link: https://github.com/ValveSoftware/gamescope

or just ask SteamOS https://github.com/ValveSoftware/SteamOS

if gamescope, the [Josuhua](https://github.com/Joshua-Ashton) would know 
then [Pierre-Loup Griffais](https://twitter.com/Plagman2/with_replies) another SteamOS coder

**Comment by wyyadd** [comment]
Yes, flickering issue is so annoying. I will try my best to fix it.

**Comment by ahjolinna** [comment]
I just remember, there is something called [Decky(-loader)](https://decky.xyz/) I was wondering you might use/support it or look into how to do something similar to run within gaming-mode
